.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,12 @@
+# This workflow runs e2e tests as a Github Action for every pull request created against this repo
```

Comment:
Is this comment providing any information that isn't clear from a quick scan of this file? It's technically already out of sync with the workflow config, since it runs both on PR and nightly.

Reply:
you are right. I have removed the comment